test

```diff
@@ -224,6 +237,43 @@ def print_tests(results):
           .format(i, output[:-1])
       )
 
+class SpinnerController:
```

Comment:
Fine. This is really a better abstraction. Thanks. We need use this minimal version later on ProjectEuler. Using `pandas` was really a bad idea. At least the people whose use daily python is really painful for not python programmers install this bullshit bloated with of C codes (although all the library is a piece of holy powers)